workflow-service/src/main/java/com/redhat/parodos/workflow/definition/service/WorkFlowDefinitionServiceImpl.java

```diff
@@ -194,6 +230,8 @@ public void saveWorkFlowChecker(String workFlowTaskName, String workFlowCheckerN
 							.findFirstByCheckWorkFlow(checkerWorkFlowDefinitionEntity))
 					.orElse(WorkFlowCheckerMappingDefinition.builder().checkWorkFlow(checkerWorkFlowDefinitionEntity)
 							.cronExpression(workFlowCheckerDTO.getCronExpression()).tasks(new ArrayList<>()).build());
+			WorkFlowTaskDefinition workFlowTaskDefinition = workFlowTaskDefinitionRepository
```

Comment:
Can you double check this? `workFlowTaskDefinition` is never used: can be this a bug?